docs/source/en/quantization/gptq.md

```diff
@@ -110,7 +119,7 @@ Only 4-bit models are supported, and we recommend deactivating the ExLlama kerne
 
 </Tip>
 
-The ExLlama kernels are only supported when the entire model is on the GPU. If you're doing inference on a CPU with AutoGPTQ (version > 0.4.2), then you'll need to disable the ExLlama kernel. This overwrites the attributes related to the ExLlama kernels in the quantization config of the config.json file.
+The ExLlama kernels are only supported when the entire model is on the GPU. If you're doing inference on a CPU with AutoGPTQ (version > 0.4.2) or GPTQModel (version > 1.4.2), then you'll need to disable the ExLlama kernel. This overwrites the attributes related to the ExLlama kernels in the quantization config of the config.json file.
```

Comment:
Let's also add a section about marlin kernel with gptqmodel

Reply:
@Qubitium

Reply:
@SunMarc @jiqing-feng  Ok I will write up a short section about Marlin. Also, Marlin in GPTQModel is actually upgraded Marlin  (ported from vLLM) and different sligtly (optimizations) from the original-Marlin in AutoGPTQ.

Reply:
Fixed